main/calculations.py

```diff
@@ -0,0 +1,147 @@
+import requests
+import json
+from datetime import datetime
+from constance import config
+from proxy_requests.proxy_requests import ProxyRequests
+
+from django.core.mail import send_mail
+from django.conf import settings
+from django.contrib.auth.models import User
+from django.contrib import messages
+
+from .models import SiteToCheck
+
+
+def send_email(message, email):
+    subject = 'Errors on my pages'
+    message = message
+    from_email = settings.EMAIL_ADDRESS
+    to_email = email
+    send_mail(
+        subject,
+        message,
+        from_email,
+        [to_email],
+        fail_silently=False,
+    )
+
+
+def my_cron_job():
```

Comment:
nie używaj nazw w stylu my_costam. Nie ma znaczenia czy to jest moje twoje czy basi :P. Ważne, że cron_job